src/Microsoft.IdentityModel.Tokens/Encryption/AuthenticatedEncryptionProvider.cs

```diff
@@ -169,8 +179,9 @@ public virtual AuthenticatedEncryptionResult Encrypt(byte[] plaintext, byte[] au
         /// <exception cref="ArgumentNullException"><paramref name="authenticatedData"/> is null or empty.</exception>
         /// <exception cref="ArgumentNullException"><paramref name="iv"/> is null or empty.</exception>
         /// <exception cref="ArgumentNullException"><paramref name="authenticationTag"/> is null or empty.</exception>
-        /// <exception cref="SecurityTokenDecryptionFailedException">signature over authenticationTag fails to verify.</exception>
-        /// <exception cref="SecurityTokenDecryptionFailedException">AES crypto operation threw. See inner exception.</exception>
+        /// <exception cref="SecurityTokenDecryptionFailedException">Thrown if the signature over authenticationTag fails to verify.</exception>
```

Comment:
nit: "over **the** authentication tag"

Reply:
resolved by: https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/commit/a62d951d0c2a5759d03372b404f345b333fcb438